package.json

```diff
@@ -43,6 +43,7 @@
     "@angular/router": "3.4.7",
     "@ng2-dynamic-forms/core": "1.3.13",
     "@ng2-dynamic-forms/ui-bootstrap": "1.3.13",
+    "@types/jquery": "^2.0.41",
```

Comment:
add jquery typings to allow passing fat arrow functions to `browser.driver.executeAsyncScript`